lib/components/shorts/shorts_video_player_screen.dart

```diff
@@ -0,0 +1,224 @@
+            }
+            setState(() {
+              isVideoPlaying = widget.controller.value.isPlaying;
+            });
+            return;
+          },
+          child: Align(
+            alignment: Alignment.center,
+            child: AspectRatio(
+              aspectRatio: 16 / 9,
+              child: VideoPlayer(widget.controller),
+            ),
+          ),
+        ),
+        FadingPlayButton(opacityLevel: isVideoPlaying ? 0.0 : 1.0, isVideoPlaying: isVideoPlaying),
+        ForwardButton(widget: widget, ref: ref, playerId: playerId, mounted: mounted),
+        Positioned(
+          right: 30,
+          bottom: 20,
+          child: GestureDetector(
+            behavior: HitTestBehavior.opaque,
+            onTap: () async {
+              setState(() {
+                widget.controller.setVolume(isVideoMuted ? 1.0 : 0.0);
+                isVideoMuted = !isVideoMuted;
+              });
+            },
+            child: Icon(
+              isVideoMuted ? Icons.volume_up_outlined : Icons.volume_off_outlined,
+              size: 40,
+            ),
+          ),
+        ),
+        VideoDescription(
+          episodeId: widget.shortsVideo.id,
+          title: widget.shortsVideo.title,
+          urlLink: '/embed/${widget.shortsVideo.id}?autoplay=true&t=${widget.controller.value.duration.inSeconds.toString()}',
+          playerId: playerId,
+        ),
+        Positioned(
+          left: 30,
+          top: 30,
+          child: IconButton(
+              color: BccmColors.label4,
+              onPressed: () {
+                context.router.pop();
+              },
+              icon: const Icon(
+                Icons.arrow_back_ios_new_outlined,
+                size: 35,
+              )),
+        ),
+      ],
+    );
+  }
+}
+
+class FadingPlayButton extends StatelessWidget {
+  const FadingPlayButton({Key? key, required this.opacityLevel, required this.isVideoPlaying}) : super(key: key);
+
+  final double opacityLevel;
+  final bool isVideoPlaying;
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: AnimatedOpacity(
+        opacity: opacityLevel,
+        duration: const Duration(seconds: 2),
+        child: Icon(
+          isVideoPlaying ? Icons.pause_sharp : Icons.play_arrow,
+          size: 50,
+          color: Colors.amber,
+        ),
+      ),
+    );
+  }
+}
+
+class ForwardButton extends StatelessWidget {
+  const ForwardButton({
+    super.key,
+    required this.widget,
+    required this.ref,
+    required this.playerId,
+    required this.mounted,
+  });
+
+  final ShortsVideoPlayerScreen widget;
+  final WidgetRef ref;
+  final String playerId;
+  final bool mounted;
+
+  @override
+  Widget build(BuildContext context) {
+    return Positioned(
+      right: 20,
+      bottom: 80,
+      child: IconButton(
+        icon: const Icon(Icons.forward_outlined),
+        iconSize: 40,
+        onPressed: () async {
```

Comment:
This can be improved by moving the logic into the parent, in a function named "openEpisode" or something.
Then you can reuse that function for both this ForwardButton and for VideoDescription